sql-parser/src/main/java/io/crate/sql/SqlFormatter.java

```diff
@@ -817,6 +817,17 @@ static String formatSortItem(SortItem sortItem) {
             default:
                 throw new UnsupportedOperationException("unknown ordering: " + sortItem.getOrdering());
         }
+
+        switch (sortItem.getNullOrdering()) {
+            case FIRST:
+                sb.append(" NULLS FIRST");
+                break;
+            case LAST:
+                sb.append(" NULLS LAST");
+                break;
+            default:
+                break;
```

Comment:
Should we add an assertion error here instead?

Reply:
as this is the sql formatter, I'd say empty/no result is a valid option

Reply:
or do you think we should print out the default if nothing is specified? (we actually expect this option to not be defined as the SortItem has an `UNDEFINED` null ordering)

Reply:
Ah right, didn't think of the `UNDEFINED`. One option would be to make the `UNDEFINED` case explicit and add a default assertion case. But it is fine as it is as well.